### PR TITLE
fix cursor pointer on dashboard left sidebar

### PR DESCRIPTION
**Pull request by x5engine**

### CODE REVIEW

**Comment by danlipert** [comment]
Thanks for the PR @x5engine ! When issuing PRs, be sure to fill out the template explaining what changed, if theres an issue that the change is tied to, and how it was tested (a screenshot or video recording is fine, or unit tests if appropriate).

File: app/assets/v2/css/dashboard.css [hunk 1]
```diff
@@ -277,3 +277,7 @@ body {
   padding-right: 0.5em;
   vertical-align: text-bottom;
 }
+
+a.filter-tag {
+    cursor: pointer;
```
Comment (thelostone-mc) [inline app/assets/v2/css/dashboard.css]: 1 tab = 2 spaces

**Comment by thelostone-mc** [comment]
As mentioned above do fill out why this PR is being raised + screenshot !
Closing this out for now but do reopen this PR once these details have been updated ^_^

PS: left a comment on the indentation too!

File: app/assets/v2/css/dashboard.css (hunk 1 above)
Comment (x5engine) [reply app/assets/v2/css/dashboard.css]: You use spaces instead of tabs?

**Pull request closed without merging**